osu.Game/Localisation/SkinComponents/SkinnableComponentStrings.cs

```diff
@@ -79,6 +79,16 @@ public static class SkinnableComponentStrings
         /// </summary>
         public static LocalisableString TextColourDescription => new TranslatableString(getKey(@"text_colour_description"), @"The colour of the text.");
 
+        /// <summary>
+        /// "Text weight"
+        /// </summary>
+        public static LocalisableString TextWeight => new TranslatableString(getKey(@"text_weight"), @"Text weight");
+
+        /// <summary>
+        /// "The weight of the text."
+        /// </summary>
+        public static LocalisableString TextWeightDescription => new TranslatableString(getKey(@"text_weight_description"), @"The weight of the text.");
```

Comment:
would get rid of this as well because this is zero new information over the label